apps/backoffice-v2/src/main.tsx

```diff
@@ -11,6 +11,10 @@ import { Router } from './Router/Router';
 import { env } from './common/env/env';
 import './i18n';
 import './index.css';
+import dayjs from 'dayjs';
+import advancedFormat from 'dayjs/plugin/advancedFormat';
+
+dayjs.extend(advancedFormat);
```

Comment:
what would happen for Chinese

Reply:
`dayjs.format` controls the format for all locales.